CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 __New features__
 - Add the ability to send APN and FCM push notifications. Also adds the ability to query _PushStatus ([#371](https://github.com/parse-community/Parse-Swift/pull/371)), thanks to [Corey Baker](https://github.com/cbaker6).
 - Add ParseSchema, ParseCLP, and ParseFieldOptions. Should only be used when using the Swift SDK on a secured server ([#370](https://github.com/parse-community/Parse-Swift/pull/370)), thanks to [Corey Baker](https://github.com/cbaker6).
-- Add ParseInstagram authentication ([#372](https://github.com/parse-community/Parse-Swift/pull/372)), thanks to [Ulaş Sancak](https://github.com/rocxteady).
+- Add ParseInstagram authentication ([#372](https://github.com/parse-community/Parse-Swift/pull/372)) and ParseSpotify authentication ([#375](https://github.com/parse-community/Parse-Swift/pull/375)) thanks to [Ulaş Sancak](https://github.com/rocxteady).
```

Comment:
Can you add this a separate line? You can copy/paste/modify your previous line and swap out Instagram for Spotify.

Reply:
I made the changes. Thank you.

Reply:
Can you put your change at the top of the list? You only need to move your latest as I've moved your previous one in an open PR.

Reply:
Done. Thanks.

Reply:
Can you please check my update when you have time @cbaker6? Thank you.